### PR TITLE
fix(testloop): set epoch length in testloop

### PR DESCRIPTION
**Pull request by staffik**
Some resharding tests failed if an RPC node was used to submit transactions.

It worked without RPC because `client0` was used by default, and it turned out this client was a chunk validator for the child shard after resharding.
It did not work with RPC (or any client other than `client0`) because a transaction near epoch boundary was not forwarded to `client0`. It was not forwarded because `config.epoch_length` was not properly set in testloop.
It has default value `10`, while resharding tests use `6`. That caused `get_next_epoch_id_if_at_boundary` to return different result that expected. In consequence, transaction was not sent to `client0`, which was a chunk producer for next epoch.

### CODE REVIEW

File: integration-tests/src/test_loop/builder.rs [hunk 1]
```diff
@@ -503,6 +503,7 @@ impl TestLoopBuilder {
         let genesis = self.genesis.as_ref().unwrap();
         let epoch_config_store = self.epoch_config_store.as_ref().unwrap();
         let mut client_config = ClientConfig::test(true, 600, 2000, 4, is_archival, true, false);
+        client_config.epoch_length = genesis.config.epoch_length;
```
Comment (staffik) [inline integration-tests/src/test_loop/builder.rs:506]: actual fix

File: integration-tests/src/test_loop/tests/resharding_v3.rs [hunk 2]
```diff
@@ -1146,8 +1144,7 @@ fn test_resharding_v3_yield_resume() {
             vec![account_in_left_child, account_in_right_child],
             ReceiptKind::PromiseYield,
         ))
-        // TODO(resharding): test should work without changes to num_rpcs and track_all_shards
-        .num_rpcs(0)
+        // TODO(resharding): test should work without changes to track_all_shards
         .track_all_shards(true)
```
Comment (Trisfald) [inline integration-tests/src/test_loop/tests/resharding_v3.rs:1148]: We still have the issue of `track_all_shards` must be `true`, right?

**Review by Trisfald — APPROVED** [review]
amazing 🎄

File: integration-tests/src/test_loop/tests/resharding_v3.rs (hunk 2 above)
Comment (staffik) [reply integration-tests/src/test_loop/tests/resharding_v3.rs:1148]: yes

**Pull request merged**